src/app/components/itenary/itenary.component.html

```diff
@@ -1,3 +1,44 @@
+<div class="w-full flex flex-col">
+  <nz-tabset>
+    <nz-tab nzTitle="Dec 29"></nz-tab>
+    <nz-tab nzTitle="Dec 30"></nz-tab>
+    <nz-tab nzTitle="Dec 31"></nz-tab>
+    <nz-tab nzTitle="Jan 1"></nz-tab>
+    <nz-tab nzTitle="Jan 2"></nz-tab>
+    <nz-tab nzTitle="Tab 3"></nz-tab>
+    <nz-tab nzTitle="Tab 1"></nz-tab>
+    <nz-tab nzTitle="Tab 2"></nz-tab>
+    <nz-tab nzTitle="Tab 3"></nz-tab>
+    <!-- These will be dynamically generated with a for loop once I received days data from the db -->
+  </nz-tabset>
+
+  <div class="flex justify-between items-center mb-4">
+    <div>
+      <h2 class="text-black text-xl font-bold">22 April</h2>
+      <div class="flex items-center text-dark-blue">
+        <ng-icon name="matLocationOn" class="text-xl"></ng-icon>
+        <p class="text-sm">add Location</p>
+      </div>
+    </div>
+    <button
+      class="bg-black size-10 flex justify-center items-center rounded-lg shadow-2xl self-end"
+    >
+      <ng-icon name="matEdit" class="text-white text-2xl"></ng-icon>
+    </button>
+  </div>
+</div>
+
+<app-event></app-event>
+<app-event></app-event>
+<app-event></app-event>
+<app-event></app-event>
+<app-event></app-event>
```

Comment:
are these a set amount? consider either a loop or setting this within the component itself so that you don't repeat it like this

Reply:
These will be dynamically made once I have data, I just wanted to see what it will look like